docs/authentication/session-management.mdx

```diff
@@ -0,0 +1,896 @@
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development that involves securely managing user sessions. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically access tokens. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is digitally signed and contains information about what resources can be accessed and for how long.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of. You can create a token, introspect a token, revoke a token, and retrieve active tokens.
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### Create a Token
+
+To request tokens for an app programmatically, send a request to the app's token request API.
+
+These requests follow the OAuth and OIDC protocols.
+
+#### Token Request URL and Endpoints
+
+The token request API URL is based on the Beyond Identity tenant, realm, and application for which a token is being requested.
+
+The API host name is prefixed with "auth-us" or "auth-eu", depending on the region of the tenant.
+
+:::note  
+Note that the host name for token requests is distinct from the Beyond Identity Management API, which uses "api-us" or "api-eu".  
+:::
+
+There are two main endpoints for requesting tokens:
+
+The /authorize endpoint:
+
+```http
+https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/authorize
+```
+
+and the /token endpoint:
+
+```http
+https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/token
+```
+
+where:
+
+- '{tenant_id}' is the [ID of the tenant](/docs/next/find-tenant-id) in which the app is configured
+
+- '{realm_id}' is the **Realm Id** of the realm in which the application is configured
+
+- '{application_id}' is the **Application ID** of the app
```

Comment:
It might be nice to use the table format here that we've been using for recent docs for consistency

Reply:
Done 👍